cli/cmd/uninstall.go

```diff
@@ -36,9 +36,16 @@ var uninstallCmd = &cobra.Command{
 		}
 		ctx := cmd.Context()
 
+		errMessage := fmt.Errorf("Odigos is not installed in the cluster")
 		ns, err := resources.GetOdigosNamespace(client, ctx)
 		if err != nil {
-			ns = consts.DefaultNamespace
+			if errors.IsNotFound(err) {
+				fmt.Println(errMessage)
+			} else {
+				ns = consts.DefaultNamespace
```

Comment:
this is redundant since we `os.Exit` right after

Reply:
- I just found that my if condition never triggers, so i removed it